amundsen_application/static/js/components/Feedback/FeedbackForm/BugReportFeedbackForm/tests/index.spec.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import { shallow } from 'enzyme';
 
 import AbstractFeedbackForm, { FeedbackFormProps } from 'components/Feedback/FeedbackForm';
-import { SendingState } from 'components/Feedback/types';
+import { SendingState } from 'interfaces';
```

Comment:
I find there's a benefit for components to only need to know that they are imported a shared interface from `interfaces` without needing to know what file it lives in. I think this is especially helpful as we figure out what we're doing and what is best -- we have free range to restructure and rearrange `interfaces` without needing to make component changes. 

The way I see us resolving our spaghetti import problem is to move away from importing from file by file and think about how we can better bundle those files and import from that bundle. I'm not convinced what I've set up is perfect, however it is set up in a way that let's us experiment and switch things up with less burden.

Reply:
Personally I would prefer to be more explicit with my imports, but I think our codebase is small enough to avoid any collisions in terms of ambiguous naming and collisions. I'm okay with trying this out and revisiting if it ever becomes an issue.